### PR TITLE
Pin to Rust nightly, after all

### PR DESCRIPTION
**Pull request by kim**
libp2p currently requires it.

### CODE REVIEW

File: .buildkite/pipeline.yaml [hunk 1]
```diff
@@ -1,38 +1,14 @@
...
-  concurrency: 1
-
-- label: "Nightly Rust"
-  commands:
-  - "cargo test --all"
-  agents:
-    production: "true"
-    platform: "linux"
-  env:
+    #DOCKER_IMAGE: "gcr.io/opensourcecoin/radicle-link-build@sha256:"
```
Comment (FintanH) [inline .buildkite/pipeline.yaml]: Intentional?

Comment (kim) [reply .buildkite/pipeline.yaml]: Yah, waiting for that sha to be mined

**Comment by kim** [comment]
Remaining question: do you prefer freedom of artistic expression or should `clippy` also pass on CI? @FintanH @massimiliano-mantione

File: .buildkite/pipeline.yaml (hunk 1 above)
Comment (FintanH) [reply .buildkite/pipeline.yaml]: Ya, saw the next commit as soon as I commented :sweat_smile:

**Comment by FintanH** [comment]
@kim: Minimising bike shedding would be good. If someone wants to express themselves then they can add a lint rule and argue for it in review :)

**Pull request merged**